infra/ansible/files/authorized_keys

```diff
@@ -0,0 +1,10 @@
+# Anton
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDckh8AO6+am3nFudUrRgmkqfXn2al6pSSAvnKIzHAnRrkhozXoO6yXpA+oei4wNglpIr0GO3FgZGkuddSZfxAcIaGdCCEj3LKXkIr7Oc+DjpFT/LjECMaeVncsK6c9DrDtE/Yjg76BfxkM6gP5VP773lrioFSnzytZPaJE/ZDc3wywh3K345D4LPufuP3b447Q/k51ZxsPHuOhh/201Gpp1ZjX6mk1gPDRBuX5cWyU6MBNt7M/r5/xfpM144BXz50ARVZGv8AGVAhubn2VYpMaq8vM4rsS8AACjJGQ/GlkZDuH5YFflsJ4D0UK1jbrvgt/16Az40GaY8gX2QiMFDb9
```

Comment:
One concern I have with baking in the ssh keys directly is that if a non-PL community member wants to set up testground on their own AWS account, they might not see this and might accidentally grant access.

Reply:
I totally agree!

I think we should split the parts that are open source about our infrastructure, from the parts that are specific to PL. At the same time we want our infrastructure to be defined in code and under version control, so ultimately we need this data somewhere in a repo.

For now I consider `ipfs/testground` to be that repo (as visible in this PR), but maybe that's not right. I am open to suggestions on how and where to define the PL-specific parts.

In the past we've started with private repos, and then open sourced bits and pieces, but I have found this approach a bit lacking, because ultimately the private repos were always ahead of the public open-sourced versions.

I don't know what's the best way to do that.

Reply:
My concern with PL-specific infrastructure code is that it might not be private, but it will leak information about PL infrastructure, which could be a security concern. I think we spoke briefly recently about whether to introduce private repos for Testground infrastructure, and decided not to. I think it would be worth considering this here as well - while these SSH keys are not private, I don't see any benefit in them being in a public repo. There are other such cases - for example our DNS records.

Reply:
IPFS-Cluster has this with its ansible repo, see https://github.com/hsanjuan/ansible-ipfs-cluster for the opensource stuff, and then there is the PL-specific implementation of such, which is private.

Reply:
Another way to do this is to have an ansible role that slurps up github usernames and then uses the `github.com/<username>.keys` endpoint to gather the public keys. This way the users are just a variable in the host_vars, which makes it easy for other community members to swap them out.

Reply:
@jimpick @lanzafame I went back on the idea to have static SSH keys in this repo. I still think we should do it for the common infrastructure that the team maintains, but decided that this is not the right place for it.

I pretty much reverted the key configuration as @jimpick had done it in the initial PR.